templates/download/desktop/index.html

```diff
@@ -65,96 +73,153 @@ <h2>Ubuntu {{ releases.latest.full_version }}</h2>
-                alt="",
-                width="60",
-                height="60",
-                hi_def=True,
-                loading="lazy",
-                attrs={"class": "p-heading-icon__img"},
-                ) | safe
-          }}
-          <h3 class="p-heading-icon__title">From Windows</h3>
-        </div>
-      </div>
-      <p>If you're using Windows 10 or any computer with a 64-bit processor, we recommend the 64-bit download.</p>
-      <ul class="p-list">
-        <li class="p-list__item"><a href="/tutorials/tutorial-burn-a-dvd-on-windows">How to burn a DVD on Windows</a></li>
-        <li class="p-list__item"><a href="/tutorials/tutorial-create-a-usb-stick-on-windows">How to create a bootable USB stick on Windows</a></li>
-      </ul>
+    <div class="col-4 u-vertically-center u-align--center">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/ea34f006-Multipass+logomark_rgb.svg",
+        alt="",
+        width="150",
+        height="150",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
     </div>
-
-    <div class="col-4 p-card">
-      <div class="p-heading-icon--small">
-        <div class="p-heading-icon__header">
-          {{
-            image(
-                url="https://assets.ubuntu.com/v1/4239507e-picto-mac.svg",
-                alt="",
-                width="60",
-                height="60",
-                hi_def=True,
-                loading="lazy",
-                attrs={"class": "p-heading-icon__img"},
-              ) | safe
-          }}
-          <h3 class="p-heading-icon__title">From macOS</h3>
-        </div>
-      </div>
-      <p>Most Macs with Intel processors will work with either 64-bit or Mac images. If the 64-bit image doesn't work, try the Mac image.</p>
-      <ul class="p-list">
-        <li class="p-list__item"><a href="/tutorials/tutorial-burn-a-dvd-on-macos">How to burn a DVD on macOS</a></li>
-        <li class="p-list__item"><a href="/tutorials/tutorial-create-a-usb-stick-on-macos">How to create a bootable USB stick on macOS</a></li>
-      </ul>
+  </div>
+  <div class="u-fixed-width">
+    <hr class="p-separator">
+  </div>
+  <div class="row">
+    <div class="col-4 u-align--center">
+      {{ image (
+        url="https://assets.ubuntu.com/v1/c4f35e06-products-hero-ubuntu.svg",
+        alt="",
+        width="140",
+        height="140",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+      {{ image (
+        url="https://assets.ubuntu.com/v1/66c99aa0-windows-logo-blue.svg",
+        alt="",
+        width="112",
+        height="115",
+        hi_def=True,
+        loading="lazy"
+        ) | safe
+      }}
+    </div>
+    <div class="col-8">
+      <h3>Access the Linux Terminal on Windows with Ubuntu WSL</h3>
+      <p>For a more integrated environment, activate Windows Subsystem for Linux (WSL) to run Linux applications and workflows while developing cross-platform on your Windows machine.</p>
```

Comment:
Missing the "You can download Ubuntu directly from the Microsoft store" sentence